### DISCUSSION TITLE
when I use“Xtime2 = extract_features(df131_new, column_id='name', column_sort='time')”，there is an error

### DISCUSSION THREAD

**Discussion by Lixiao1273** [blue-yonder/tsfresh#1014] [Q&A]
RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

**Comment by nils-braun** [comment]
Hi!
Have you tried adding the `if __name__ == '__main__':` as mentioned in the error message? Does it still fail?

**Reply by Lixiao1273** [reply]
Thank you very much. What you said solved my problem. As I am a beginner, I made this mistake. Finally, thank you very much for solving my question.
